libs/api/src/lib/graphql/comment/comment.public-queries.ts

```diff
@@ -99,27 +111,39 @@ export const getPublicCommentsForItemById = async (
       include: {
         revisions: {orderBy: {createdAt: 'asc'}},
         ratings: true,
-        overriddenRatings: true
+        overriddenRatings: true,
+        tags: {include: {tag: true}}
       },
       orderBy: {
         createdAt: 'asc'
       }
     })
   ])
 
-  const commentsWithRating = comments.map(({ratings, ...comment}) => ({
+  const commentsWithRating: CommentWithTags[] = comments.map(({ratings, tags, ...comment}) => ({
     ...mapCommentToPublicComment(comment),
-    calculatedRatings: getCalculatedRatingsForComment(answers, ratings)
+    calculatedRatings: getCalculatedRatingsForComment(answers, ratings),
+    tags: tags || []
   }))
 
+  let sortedComments = commentsWithRating
   if (sort === PublicCommentSort.Rating) {
     if (order === SortOrder.Ascending) {
-      return sortCommentsByRating(ascend)(commentsWithRating)
+      sortedComments = sortCommentsByRating(ascend)(sortedComments)
+    } else {
+      sortedComments = sortCommentsByRating(descend)(sortedComments)
     }
-
-    return sortCommentsByRating(descend)(commentsWithRating)
   }
 
-  // no sorting needed as comments already come sorted by creation
-  return commentsWithRating
+  const topComments = sortedComments.filter(comment =>
+    comment.tags.some(({tag}) => tag.tag === 'Top Kommentar')
```

Comment:
We talked in the meeting on the 12th that this should not be a new property on the comment and not use magic tags for this.

Reply:
can you double check this with Michael because I talked with him on Monday and he said that it's fine as it's done now (using Top Kommentar)